timemachine/md/minimizer.py

```diff
@@ -551,6 +553,7 @@ def local_minimize(
     verbose: bool = True,
     assert_energy_decreased: bool = True,
     restraint_k: Optional[float] = None,
```

Comment:
Preference is to avoid support for `None` - should be equivalent to `restraint_k: Optional[float] = 0.0`

Reply:
Defaults to 0.0 and remove optional in https://github.com/proteneer/timemachine/pull/1421/commits/20c2ee2b5363a5fe893bce3da4f1db3436a41a77